h/static/scripts/annotation-metadata.js

```diff
@@ -70,30 +70,55 @@ function isNew(annotation) {
   return !annotation.id;
 }
 
+/** Return `true` if the given annotation is a page note, `false` otherwise. */
+function isPageNote(annotation) {
+  if (annotation && !annotation.deleted) {
+    if (isReply(annotation)) {
+      return false;
+    }
+    if (annotation.target && (annotation.target.length === 0 || !annotation.target[0].selector)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+/** Return `true` if the given annotation is a top level annotation, `false` otherwise. */
+function isAnnotation(annotation) {
```

Comment:
Couple of comments here:

1. You can simplify the `isPageNote()` function, conceptually, to:

```
isPageNote = annot => !isAnnotation(annot) && !isReply(annot)
```

2. Avoid checks for whether an argument is null/undefined unless there is a clear reason _why_ the parameter would ever be null. Otherwise you have introduced an extra path through your code and until we have better tools, it is difficult to find out after the fact whether such checks are necessary.